logback-android/src/main/java/ch/qos/logback/core/android/AndroidContextUtil.java

```diff
@@ -96,7 +96,7 @@ public String getMountedExternalStorageDirectoryPath() {
     String state = Environment.getExternalStorageState();
     if (state.equals(Environment.MEDIA_MOUNTED) ||
         state.equals(Environment.MEDIA_MOUNTED_READ_ONLY)) {
-      path = absPath(Environment.getExternalStorageDirectory());
+      path = this.context != null ? this.context.getExternalFilesDir(null).getAbsolutePath() :  null;
```

Comment:
This code is similar to `getExternalStorageDirectoryPath()`, so we should call it here.

Reply:
Seems reasonable :-)